src/bacnet/bacenum.h

```diff
@@ -1598,8 +1598,8 @@ typedef enum BACnet_Services_Supported {
     SERVICE_SUPPORTED_PRIVATE_TRANSFER = 18,
     SERVICE_SUPPORTED_TEXT_MESSAGE = 19,
     SERVICE_SUPPORTED_REINITIALIZE_DEVICE = 20,
-    SERVICE_SUPPORTED_WHO_AM_I = 47,
-    SERVICE_SUPPORTED_YOU_ARE = 48,
+    SERVICE_SUPPORTED_WHO_AM_I = 47,    /* 2016 + bz    22*/
```

Comment:
Please use the same form that BACnetObjectType enumeration is using for the addendum comment:
```
/* Addendum 135-2016bz */
```

Reply:
Thanks Steve. The comments are fixed in my latest commit.